File: Array and String/Two_sum_II.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        int lowIndex = 0;                                                               
        int highIndex = numbers.size()-1;
        vector<int>v;
        while(lowIndex < highIndex)
        {
            int temp = numbers[lowIndex] + numbers[highIndex];                                 
                if(temp == target)
                {
                    v.push_back(lowIndex+1);
                    v.push_back(highIndex+1);
                    return v;
                }
        if(temp < target)                                                                 
            lowIndex++;
        else
            highIndex--;
        }  
        return v;
    }
};

File: Array and String/Two_sum_II.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        lowIndex = 0                                                               
        highIndex = len(numbers)-1
        while (lowIndex < highIndex):
            temp = numbers[lowIndex] + numbers[highIndex]                                 
            if temp == target:
                return [lowIndex+1, highIndex+1]
            if temp < target:                                                                
                lowIndex+=1
            else:
                highIndex-=1
        return []
        